app/waf/src/models/pool.model.ts

```diff
@@ -30,6 +25,15 @@ export class Pool extends Entity {
     itemType: 'string',
     required: false,
     default: [],
+    schema: {
```

Comment:
Monitors will not be a column any longer. You may implement it in your next many-to-many relation pr. Needn't to implement this property any more.

Reply:
Yes, I will change it, in next many-to-many PR. Here is just for code consistency.